.github/workflows/ruby.yml

```diff
@@ -14,6 +14,7 @@ on:
     branches: ['master']
   schedule:
     - cron: '0 0 * * 0'
```

Comment:
Also, consider removing this scheduling. See the notes from active-record_acts-as